### ISSUE TITLE
Write "Collaborating" section in README.md or add COLLAB.md

### ISSUE THREAD

**Issue by tendiebawse** [TheWallStreetAnalytics/wallstreetanalytics#27]
With such a flurry of activity I think we should update README.md or create a COLLAB.md to have links to relevant places where people can get involved (GitHub, Discord, etc.).   We should then provide some notes about how and where discussions will take place so we can make decisions.  We've got to codify #10 into a document.

**Comment by Mantosh** [comment]
Good idea. Would you mind suggesting an example or two?

**Comment by tendiebawse** [comment]
Yeah, happy to write something up.  Where are people working right now on discord?  Are there any other communications tools/channels people are working in?  Sub-reddits or posts that people are still discussing in?

**Comment by MattSmilin** [comment]
There's a discord, but I cannot find an invite, nor I have permissions to create invites. Let me get back to you on this.
